Documentation/ceph-cluster-crd.md

```diff
@@ -311,7 +314,10 @@ Here are the current minimum amounts of memory in MB to apply so that Rook will
 * `mds`: 4096MB
 * `rbdmirror`: 512MB
 
-Rook does not enforce any minimum limit nor request on the prepare OSD pod. We have seen that pod taking up to 50 MB, but depending on the OSD scenario, that value might need to be higher. So setting a value to at least 50 MB should be a good start, but 100 MB might be safer.
+Rook does not enforce any minimum limit nor request on the following:
+
+* prepare OSD pod: we have seen that pod taking up to 50 MB, but depending on the OSD scenario, that value might need to be higher. So setting a value to at least 50 MB should be a good start, but 100 MB might be safer.
```

Comment:
Suggestion: "This pod commonly takes up to 50MB, but depending on the OSD scenario may need more memory. 100MB would be more conservative."